packages/go_router/lib/src/route.dart

```diff
@@ -263,6 +290,11 @@ class GoRoute {
   /// for a complete runnable example.
   final GoRouterRedirect redirect;
 
+  /// An optional key specifying which Navigator to display this route's screen
```

Comment:
A new line between the first sentence and next

Reply:
886bee7829c4b5fc701bb83f52653ffec12b0d2e